spec/features/web_helpers.rb

```diff
@@ -0,0 +1,8 @@
+def sign_in_and_play 
```

Comment:
This is good automation of the feature test! This helps to keep your code neat in your other feature file, e.g. player_spec.rb